### PR TITLE
fix(inheritance): Add elements not found in schema

### PR DESCRIPTION
**Pull request by tymokvo**
Re: pollination/pollination-server#194

The `RoleEnum` is not found from the call to `pydantic.schema.schema` and thus isn't added to the `enums` dictionary by the loop on line 320 in `inheritance.py`. This adds the missing elements to the `enums` dictionary.

I've tested this interactively against `pollination-server`. The generated JSONs with this change are [here](https://storage.cloud.google.com/lbt-blobs/misc-dev-blobs/pydantic-openapi-helper/openapi_mapper.json). I'm not sure if this is how it needs to work for C# code generation necessarily, though.

### CODE REVIEW

File: pydantic_openapi_helper/inheritance.py [hunk 1]
```diff
@@ -304,6 +312,10 @@ def class_mapper(models, find_and_replace=None):
 
     # add enum classes to mapper
     schemas = get_schemas_inheritance(models)
+
+    missing_items = {k: v for k,
+                     v in mapper.items() if k not in schemas.keys()}
```
Comment (mostaphaRoudsari) [inline pydantic_openapi_helper/inheritance.py]: This is probably more 'pythonic'!

```suggestion
    missing_items = {k: v for k,
                     v in mapper.items() if k not in schemas}
```

**Review by mostaphaRoudsari — APPROVED** [review]
It looks good to me but @MingboPeng should test the generated OpenAPI files before we merge this in.

**Comment by tymokvo** [comment]
@MingboPeng let me know if you need anything from me to test this

**Comment by mostaphaRoudsari** [comment]
I added @MingboPeng as a reviewer so it pops up on his GitHub page!

**Comment by MingboPeng** [comment]
Hi @tymokvo, I don't have access to: https://storage.cloud.google.com/lbt-blobs/misc-dev-blobs/pydantic-openapi-helper/openapi_mapper.json
image

Meanwhile, could you also please provide me the openapi_inheritance.json as well?

**Comment by MingboPeng** [comment]
Nevermind, I found them from Google Cloud

**Comment by MingboPeng** [comment]
openapi_inheritance.json still doesn't include **RoleEnum**
image

enums in mapper.json are duplicated:
image

**Review by MingboPeng — CHANGES_REQUESTED** [review]
Thanks, please see my above comments

**Comment by MingboPeng** [comment]
Hi @tymokvo please let me know if you need more info to fix this issue. Thanks.

**Comment by tymokvo** [comment]
Hi @MingboPeng so, just to clarify, `RoleEnum` needs to be in `openapi_inheritance.json` in the same way that (e.g.) `OrganizationRoleEnum` is?

**Comment by mostaphaRoudsari** [comment]
Hi @tymokvo, Yes all the Enums should show up under that list

**Comment by MingboPeng** [comment]
Just to repot the new SortEnum is also missing.

**Comment by tymokvo** [comment]
I think this is closed by https://github.com/pollination/pollination-server/pull/290, closing. Re-open if not.

**Pull request closed without merging**